migrations/2025-06-19-170656_alter_payout_primary_key/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE payout_attempt DROP CONSTRAINT payout_attempt_pkey;
+ALTER TABLE payout_attempt ADD PRIMARY KEY (payout_attempt_id, merchant_id);
```

Comment:
Nit: This would have to match the order of the columns used in the query as well, as in, the DB queries would also have to query using (payout_attempt_id, merchant_id), not (merchant_id, payout_attempt_id), as the index would not be used in the latter case (as far as I understand).